pymc/distributions/continuous.py

```diff
@@ -995,6 +995,11 @@ def dist(
 
         return super().dist([mu, lam, alpha], **kwargs)
 
+    def get_moment(rv, size, mu, lam, phi):
```

Comment:
You will need to broadcast mu to the shape of lam and phi, and add test conditions that would fail if you were not to do that. For instance by making mu a scalar, and lam or phi vectors (and size None).


```suggestion
    def get_moment(rv, size, mu, lam, phi):
        mu, _, _, = at.broadcast_arrays(mu, lam, phi)
```

Reply:
I added broadcasting, for the tests, would it be fine to go over 4 tests or should I try to blend it into the existing ones?

Reply:
It's fine to go over 4 if they are testing different things, specially with a distribution that has 3 parameters

Reply:
I added 3 more tests to check different parameters.

Reply:
The tests do not cover the cases where `at.broadcast_arrays` is needed. You can confirm this by commenting out the line in `get_moment` and check that all tests still pass. You need to choose a combination of parameters that would fail without this line.

The following seem to do the trick:
```python
(1, None, np.ones(5), None, np.full(5, 1)),
(3, np.full(5, 2), None, None, np.full(5, 3)),
```

Besides this, it should be enough to include 2/3 tests more. More than 5 is probably too much.

Let me know if you have any questions

Reply:
Got it. I thought changing it to vector would do the trick. Thanks for the guidance. Updated the tests as recommended.